Typo fixes and translate to enGB (#6485)
ppy/osu-wiki: wiki/Tournament_Drawings/en.md

There are 4 buttons in the control panel, let's go over them:

- **Begin random**
  - This will start the randomisation process, causing the flags on the screen to scroll.
- **Stop random**
  - This will stop the randomisation process, causing the scrolling to slow down until eventually stopping and centring on a flag in the middle of the screen.
- **Reload**
  - This will reload the `drawings.txt` file.
- **Reset**
  - This button should rarely be used. It is destructive and will reset the results of the drawings process.

Press the `Begin random` and the `Stop random` buttons. When the scrolling stops on a flag at the centre of the screen, go back to the osu! folder and notice that there is now an additional file created:

```

> drawings_results.txt  

Reviewer: didn't the previous option work? why put the file name on a separate line?

Reviewer 2: agree https://github.com/ppy/osu-wiki/pull/6485#discussion_r762085383

> Press the `Begin random` and the `Stop random` buttons. When the scrolling stops on a flag at the centre of the screen, go back to the osu! folder and notice that there is now an additional file created:
> 
> ```
> drawings_results.txt  
> ```

Reviewer 2: ```suggestion
Press the `Begin random` and the `Stop random` buttons. When the scrolling stops on a flag at the centre of the screen, go back to the osu! folder and notice that an additional file has been created, called `drawings_results.txt`.
```
